src/components/PluginWallet/PluginWallet.tsx

```diff
@@ -0,0 +1,95 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React from 'react';
+
+import { ReactNode } from 'react';
+import { Button } from '../Button';
+import { FiArrowUpRight } from 'react-icons/fi';
+
+export interface PluginWalletProps {
+  isActive: boolean;
+  title: string;
+  iconActive: ReactNode;
+  iconInactive: ReactNode;
+  onClickActive: () => void;
+  onClickInactive: () => void;
+}
+
+export interface ActivePluginProps {
+  title: string;
+  iconActive: ReactNode;
+  onClickActive: () => void;
+}
+
+export interface InactivePluginProps {
+  title: string;
+  iconInactive: ReactNode;
+  onClickInactive: () => void;
+}
+
+export function ActivePlugin(props: ActivePluginProps) {
+  const { title, iconActive, onClickActive } = props;
+
+  return (
+    <>
+      {iconActive}
+      <div className="w-full py-6 text-f-primary bg-secondary flex flex-col items-center">
```

Comment:
```suggestion
      <div className="w-full py-6 text-f-primary bg-secondary m-auto">
```

Can you test with this proposition ? And after ping me if it work. 🤔

Reply:
doesn't work:
![image](https://github.com/massalabs/ui-kit/assets/91454320/821fa3eb-4a1e-4283-9283-d9cf956b8b91)

Reply:
ok